app/views/events/addresses/_address.xml.haml

```diff
@@ -7,6 +7,8 @@
     %address_line_3= address.address_3
   %location_city_name= address.city
   %location_state_code= address.state
+  - if EnrollRegistry[:enroll_app].setting(:county_fips_code_enabled).item && address.fetch_county_fips_code.present?
```

Comment:
one last comment, can you add spec for this change
refer here(spec/views/events/v2/addresses/_address.xml.haml_spec.rb) if we don't have the spec for address.xml.haml

Reply:
Added spec